eloquent/distribution.yaml

```diff
@@ -1415,13 +1415,13 @@ repositories:
     source:
       type: git
       url: https://gitlab.com/micro-ROS/ros_tracing/ros2_tracing.git
-      version: master
+      version: 0.2.10
```

Comment:
Since this is a tag `test_commits` and `test_pull_requests` will need to be made explicitly false.

Reply:
Done!